src/target/riscv/riscv-013.c

```diff
@@ -1709,13 +1738,16 @@ static target_addr_t sb_read_address(struct target *target)
 	RISCV013_INFO(info);
 	unsigned sbasize = get_field(info->sbcs, DMI_SBCS_SBASIZE);
 	target_addr_t address = 0;
+	uint32_t v;
 	if (sbasize > 32) {
 #if BUILD_TARGET64
-		address |= dmi_read(target, DMI_SBADDRESS1);
+		dmi_read(target, &v, DMI_SBADDRESS1);
```

Comment:
another place dmi-read result is not checked

Reply:
This function doesn't return an error code either.